package.json

```diff
@@ -13,7 +13,8 @@
     "pretest": "tslint --project ./src/ && tslint  ./test/**/*.ts && npm run build",
     "test": "nyc mocha",
     "prepare": "npm run build",
-    "kafka:start": "KAFKA_TOPICS=${KAFKA_TOPICS:-orka.example.test:1:1} HOST_IP=$(ifconfig | grep -E \"([0-9]{1,3}\\.){3}[0-9]{1,3}\" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1) docker-compose -f kafka-docker-compose.yml up"
+    "kafka:start": "KAFKA_TOPICS=${KAFKA_TOPICS:-orka.example.test:1:1} HOST_IP=$(ifconfig | grep -E \"([0-9]{1,3}\\.){3}[0-9]{1,3}\" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1) docker-compose -f kafka-docker-compose.yml up",
+    "kafka:start:ci": "KAFKA_TOPICS=${KAFKA_TOPICS:-orka.example.test:1:1} HOST_IP=$(ifconfig | grep -E \"([0-9]{1,3}\\.){3}[0-9]{1,3}\" | grep -v 127.0.0.1 | awk '{ print $2 }' | cut -f2 -d: | head -n1) docker-compose -f kafka-docker-compose.yml up -d"
```

Comment:
❓ In what does it differ from kakfa:start what's `-d` doing?

Reply:
-d is for detached in order to run the command and return. Otherwise the flow was stuck in this command and never ran the tests